### ISSUE TITLE
Pulse Nova turns off immediately when drafted on Huskar's model due to mana=0

### ISSUE THREAD

**Issue by 7forks** [ValveSoftware/Dota2-Gameplay#19447]
### Description

Basically it appears that Pulse Nova has some sort of detection mechanism that turns off automatically when the mana of the user is less than the cost or something along the line;
 
Now when PN is drafted on Huskar's model, the mana cost to life cost innate ability (blood magic) works fine, but this pulse nova check turns it off immediately on use because Huskar's mana is considered 0 at all time.
 
So basically I had to press R every 1 second to use it....the entire game...
 
Pretty sure this is not intended....

### Example Match ID (and possibly Timestamp)

7800507700 - 11:11
